client/src/test/omtLinkProvider.test.ts

```diff
@@ -0,0 +1,62 @@
+import { DocumentLink, commands, workspace, Uri, Position, Range } from 'vscode';
+import { equal, deepEqual } from 'assert';
+import { getDocUri, activate } from './helper';
+
+describe('Should make links of imports', () => {
```

Comment:
A describe should state the thing that is subjected to testing
```suggestion
describe('OMTLinkProvider', () => {
```